### PR TITLE
support preinstalled local pixi

### PR DESCRIPTION
**Pull request by ytausch**
If updating documentation:

- [x] Updated documentation in https://github.com/prefix-dev/pixi/blob/main/docs/advanced/github_actions.md as well https://github.com/prefix-dev/pixi/pull/1181

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -230,6 +230,52 @@ jobs:
         # which pixi should be absolute
         which pixi | grep -q "^/"
 
+  existing-pixi-bin:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Move pixi.toml
+        run: mv test/default/* .
+      - name: Download pixi binary
```
Comment (pavelzw) [inline .github/workflows/test.yml:239]: Would be cool to also test this on windows. You can do a custom step with `if: ${{ matrix.os != 'windows-latest' }}`

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -230,6 +230,52 @@ jobs:
...
+          echo ~/custom-existing-pixi >> $GITHUB_PATH
+      - uses: ./
+        with:
+          cache: false
+      - run: |
+          set -o pipefail
+          pixi --version | grep -q "pixi 0.14.0"
+          which pixi | grep -q ~/custom-existing-pixi/pixi
+
+  dont-use-existing-pixi-bin:
```
Comment (pavelzw) [inline .github/workflows/test.yml:255]: Checking if this works on windows would be nice.

**Review by pavelzw — COMMENTED** [review]
LGTM apart from missing windows tests. Can you add them?

**Comment by pavelzw** [comment]
WDYT of this change @ruben-arts? This would make it easier for using setup-pixi in corporate environments where the pixi URL is not easily accessible. This would enable using setup-pixi in corporate environments without the need for specifying a custom pixi url every time.

File: .github/workflows/test.yml (hunk 1 above)
Comment (ytausch) [reply .github/workflows/test.yml]: Done

File: .github/workflows/test.yml (hunk 2 above)
Comment (ytausch) [reply .github/workflows/test.yml]: Done

**Comment by ytausch** [comment]
The tests should now cover Windows and all functionality of my change.

File: .github/workflows/test.yml [hunk 3]
```diff
@@ -230,6 +230,126 @@ jobs:
         # which pixi should be absolute
         which pixi | grep -q "^/"
 
+  existing-pixi-bin:
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        ignoreReason: [none, version, version-latest, url, bin-path]
```
Comment (pavelzw) [inline .github/workflows/test.yml:237]:
```suggestion
        ignore-reason: [none, version, version-latest, url, bin-path]
```

we use kebab-case in GHA

**Review by pavelzw — APPROVED** [review]
Thanks!

File: .github/workflows/test.yml (hunk 3 above)
Comment (ytausch) [reply .github/workflows/test.yml]: fixed

**Review by ruben-arts — APPROVED** [review]
LGTM! Thanks for the contribution @ytausch

**Comment by pavelzw** [comment]
We'll release this together with #87

**Pull request merged**